ethereum/dex/view_token_prices.sql

```diff
@@ -0,0 +1,46 @@
+BEGIN;
+
+DROP MATERIALIZED VIEW IF EXISTS dex.view_token_prices;
+
+CREATE MATERIALIZED VIEW dex.view_token_prices AS (
+    WITH dex_trades AS (
+        SELECT 
+            token_a_address as contract_address, 
+            usd_amount/token_a_amount as price,
+            block_time
+        FROM dex.trades
+        WHERE 1=1
```

Comment:
what's the purpose of this line? @markusbkoch

Reply:
merely aesthetics 🙈 😄 don't mind dropping it